src/libbsdl/src/parallel.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <atomic>
+#include <thread>
+#include <vector>
+
+// Silly, simple and good enough parallel_for for the LUT
+// generation tools
+template<typename F>
+void
+parallel_for(unsigned begin, unsigned end, F c)
+{
```

Comment:
Should this be in the BSDL namespace?

Reply:
Oh, right, I was so worried about a namespace clash with other parallel_for that I forgot that this is only used in the table-building step.